less/misc.less

```diff
@@ -53,6 +53,9 @@ input, textarea {
 	visibility: hidden;
 	top: 0;
 	float: left;
+	.selectlist {
+		display: block !important;
```

Comment:
Sizer is fine. We can move back. I wasn't looking at the context of the css class.

On Feb 11, 2015 12:43 PM, Christopher McCulloh <notifications@github.com> wrote:

In less/misc.less<https://github.com/ExactTarget/fuelux/pull/1073#discussion_r24515227>:

> @@ -53,6 +53,9 @@ input, textarea {
>       visibility: hidden;
>       top: 0;
>       float: left;
> +     .selectlist {
> +             display: block !important;


If "sizer" is good enough for the js variable name, why are we deviating from that in the CSS? Now it is inconsistent and even more confusing.

—
Reply to this email directly or view it on GitHub<https://github.com/ExactTarget/fuelux/pull/1073/files#r24515227>.